src/app/mail/mail-sidebar/compose-mail/compose-mail.component.ts

```diff
@@ -326,12 +328,34 @@ export class ComposeMailComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   ngAfterViewInit() {
-    if (!this.settings.is_html_disabled) {
-      this.initializeQuillEditor();
+    this.initializeComposeMail();
+    if (this.forwardAttachmentsMessageId) {
+      if (this.editor) {
+        this.updateEmail();
+      } else {
+        setTimeout(() => {
+          this.updateEmail();
+        }, 1000);
+      }
+    }
+    this.shortcuts = getComposeMailShortcuts(this);
+    this.cdr.detectChanges();
+  }
+
+  initializeComposeMail() {
+    if (this.draftMail.is_html === null || this.draftMail.is_html) {
+      if (this.editor) {
+        this.initializeQuillEditor();
+      }
     } else {
       let content = this.mailData.content ? this.mailData.content : '';
+      if (this.editor) {
+        this.content = this.editor.nativeElement.firstChild.innerHTML;
+      }
+      this.content = this.content && this.content === '' ? content : this.content;
+
       if (this.content) {
-        content += this.getPlainText(this.content);
```

Comment:
why did you remove `+`

Reply:
during testing it was creating issues, let me see again

Reply:
test all cases with html enabled/disabled, send email, forward and reply cases for both html enabled and disabled.

Reply:
sure